module/sell.py

```diff
@@ -19,32 +20,34 @@ def sell(update: Update, context: CallbackContext) -> None:
+                    conn = create_connection(DB_PATH)
+                    if conn:
+                        rows = find(user_isbn, conn, 'Books')
+                        if rows:
+                            isbn = rows[0][0]
+                            title  = rows[0][1]
+                            authors  = rows[0][2]
+                            title_and_authors  = title + "di" + authors
+                            context.bot.send_message(chat_id, "Il libro è: \nTitolo: \"" + title_and_authors + "\"")
+                            add_item(conn, isbn, title, authors, username, price)
+                            context.bot.send_message(chat_id, "Il libro è stato aggiunto al database.")
+                            return
+                        
+                        found, soup = book_in_unict(user_isbn)
+                        if found:
+                            isbn = str(soup.findAll('td')).split("bibInfoData")[len(str(soup.findAll('td')).split("bibInfoData"))-1].split("\n")[1].split("<")[0]
+                            title  = soup.find('strong').text.split('/')[0]
+                            authors  = soup.find('strong').text.split('/')[1]
+                            title_and_authors  = soup.find('strong').text
+                            context.bot.send_message(chat_id, "Il libro è: \nTitolo: \"" + title_and_authors + "\"")
+                            add_book(conn, isbn, title, authors)
+                            add_item(conn, isbn, title, authors, username, price)
+                            context.bot.send_message(chat_id, "Il libro è stato aggiunto al database.")
+                            return
                     
-                    found, soup = book_in_unict(user_isbn)
-                    if found:
-                        isbn = str(soup.findAll('td')).split("bibInfoData")[len(str(soup.findAll('td')).split("bibInfoData"))-1].split("\n")[1].split("<")[0]
-                        title  = soup.find('strong').text.split('/')[0]
-                        authors  = soup.find('strong').text.split('/')[1]
-                        title_and_authors  = soup.find('strong').text
-                        context.bot.send_message(chat_id, "Il libro è: \nTitolo: \"" + title_and_authors + "\"")
-                        add_item(isbn, title, authors, username, price)
-                        add_book(isbn, title, authors)
-                        context.bot.send_message(chat_id, "Il libro è stato aggiunto al database.")
-                        return
-                    
-                    context.bot.send_message(chat_id, "Libro non trovato. Controlla di aver inserito correttamente l'ISBN.")
-                
+                        context.bot.send_message(chat_id, "Libro non trovato. Controlla di aver inserito correttamente l'ISBN.")
+                    else:
+                        context.bot.send_message(chat_id, "Si è verificato un problema nella lettura del database.")
```

Comment:
optional, you could save a variable witha. default content `text = 'Si è verificato un problema nella lettura del database.'` and inside the if change the text content

finally, below afterward the if ..., you'll send context.bot.send_message(text)

Reply:
I think it's a matter of style and readability